### PR TITLE
Add clean-script

### LINKED ISSUE #110: Add a clean script
Let's clean out all the old stuff on start-up. 

I think there's a webpack thing that might do this but otherwise we can just add an npm script.

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #110

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,8 +16,9 @@
     "unittest": "karma start --single-run",
     "lint": "eslint .",
     "eslint": "npm run lint",
-    "build": "npm run version-check && NODE_ENV=production webpack --verbose --display-error-details --progress --colors --config src/config/webpack.prod.config.babel.js",
-    "version-check": "node bin/version-check.js"
+    "build": "npm run clean && npm run version-check && NODE_ENV=production webpack --verbose --display-error-details --progress --colors --config src/config/webpack.prod.config.babel.js",
+    "version-check": "node bin/version-check.js",
+    "clean": "rm -rf ./dist/*.{css,js,map} ./src/webpack-assets.json"
```
Comment (tofumatt) [inline package.json]: Will this work on Windows? Think of the contributors!

**Comment by tofumatt** [comment]
r+wc

File: package.json (hunk 1 above)
Comment (muffinresearch) [inline package.json]: I've added rimraf instead.

File: package.json [hunk 2]
```diff
@@ -16,8 +16,9 @@
     "unittest": "karma start --single-run",
     "lint": "eslint .",
     "eslint": "npm run lint",
-    "build": "npm run version-check && NODE_ENV=production webpack --verbose --display-error-details --progress --colors --config src/config/webpack.prod.config.babel.js",
-    "version-check": "node bin/version-check.js"
+    "build": "npm run clean && npm run version-check && NODE_ENV=production webpack --verbose --display-error-details --progress --colors --config src/config/webpack.prod.config.babel.js",
+    "version-check": "node bin/version-check.js",
+    "clean": "rimraf './dist/*+(css|js|map)' './src/webpack-assets.json'"
```
Comment (tofumatt) [inline package.json]: Thanks :+1:

**Pull request merged**